language/move-prover/bytecode-to-boogie/test_mvir/verify-stdlib/libra_account.mvir

```diff
@@ -755,7 +790,9 @@ module LibraAccount {
         succeeds_if true;
 
     // Destroy a unique handle.
-    public destroy_handle<T: unrestricted>(handle: Self.EventHandle<T>) {
+    public destroy_handle<T: unrestricted>(handle: Self.EventHandle<T>)
+        succeeds_if true
```

Comment:
Instead of using succeeds_if, I think we can say "aborts_if false" for these kinds of conditions.